driver/src/main/java/org/neo4j/driver/internal/net/pooling/PooledSocketConnection.java

```diff
@@ -281,6 +284,12 @@ public void setResourcesHandler( SessionResourcesHandler resourcesHandler )
     }
 
     @Override
+    public long creationTimestamp()
```

Comment:
Do we need to expose `creationTimestamp` and `lastUsedTimestamp` if we have a `isValid` method?

```
when obtaining a conn from the pool
do
{
    conn = pool.dequeue()
} while(!conn.isValid()) // where we calculate timestamp
```